cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebViewHelper.java

```diff
@@ -95,6 +99,60 @@ public void run() {
         });
     }
 
+    @TargetApi(11)
```

Comment:
But all cocos2d-x should not use API > 10. If you want to do it like that, you should use reflections, such as

```java
if (api > 10)
   use reflection to invoke the API > 10
```

That's mean, there is not effect if running on devices with API 10. So could you please do it like that? Thanks.